docs/getting-started/user-guides/osx.md

```diff
@@ -249,7 +249,7 @@ Decred. Open a third terminal and go to your Decred directory. Type in
 (once again using your values for the square brackets):
 
 ```no-highlight
-~/Decred/$ ./dcrctl -u [daemon username] -P [daemon password] --wallet walletpassphrase [wallet password] 0
+~/Decred/$ ./dcrctl -u [wallet username] -P [wallet password] --wallet walletpassphrase [wallet password] 0
 ```
 
 What we are doing here is authenticating with the daemon and using the
```

Comment:
I think with the above change this should say 'authenticating with the wallet', not the daemon here.